src/disjoint_set.jl

```diff
@@ -109,15 +143,15 @@ end
     intmap::Dict{T,Int}
     internal::IntDisjointSets
 
-    function (::Type{DisjointSets{T}}){T}(xs)    # xs must be iterable
+    function (::Type{DisjointSets{T}}){T}(xs, store_size::Bool=false)    # xs must be iterable
```

Comment:
This is a bit unfortunate because there's no way to inferrably create a `DisjointSet`. But if you add a 2nd parameter then it will be.